lib/vsc/filesystem/gpfs.py

```diff
@@ -197,6 +197,8 @@ def fixup_executeY_line(self, fields, description_count):
                 self.log.raiseException(("Failed to find the initial field of the line: %s after fixup and "
                                          "splitting line into [%s, %s]") % (first_field, line, remainder))
 
+        return None
```

Comment:
Not sure if `None` or `[]` is better here :)

Reply:
Should be `[]`

Reply:
@itkovian are yo sure? today it is an implicit None

Reply:
Yes, and the code that uses it looks like

~~~~
                    fixed_lines = self.fixup_executeY_line(line, description_field_count)
                    i = fields.index((field_count, line))
                    fields[i:i + 1] = map(lambda fs: (len(fs), fs), fixed_lines)
~~~~

So a `None` will result in an error due to a wrong type. `[]` will act as expected.